### DISCUSSION TITLE
Can this daemon be compiled with Android

### DISCUSSION THREAD

**Discussion by Badger688** [COVESA/dlt-daemon#573] [Q&A] [answered]
How to build dlt-daemon so that it can run on Android OS?

**Comment by minminlittleshrimp** [comment, marked as answer]
Hello @Badger688 
thank you for your question.

For Android, dlt has a file Android.bp where we define everything for Soong build.
See: https://source.android.com/docs/setup/build

What you should do to trigger the build of a module on Android:

1. Setting up the Environment ```source build/envsetup.sh```
2. Choosing a target ```lunch product_name-build_variant```
3. Building the code: jump to the dir of dlt-daemon and run ```mm```

Ref: https://source.android.com/docs/setup/build/building

Regards
